### ISSUE TITLE
Why the https_proxy.go and http_proxy.go aren't used.

### ISSUE THREAD

**Issue by st1064870** [nfx/slrp#108]
I inspected that the https_proxy.go and http_proxy.go files don't have any usages except for testing. Is there a reasoning behind this? Will they be part of a future update?

**Comment by tungtrinh1402** [comment]
i also test many times and the certificate always fails. remote error: tls: unknown certificate.

**Comment by st1064870** [comment]
@tungtrinh1402 You have to add the CA signing certificate to your HTTPS client. It is recommended to open a new issue about it. You can check inside the ca.go file and add your custom certificate export function.

**Comment by nfx** [comment]
@st1064870 Yes, eventually the issue with validating https proxies will be fixed. It's not those two files, but rather hidden deep in the custom net.ContextDialer implementation that we use for making connections. Don't have cycles for that yet, would be good if community picks it up and makes a PR.

**Comment by nfx** [comment]
@st1064870 @tungtrinh1402 Why do you need to re-test the proxies? 

The whole idea of slrp is that you use it as your http proxy (localhost:8090) and let's re-route connections under the hood, without having to worry about checking things. 

What are your use-cases?

**Comment by nfx** [comment]
@tungtrinh1402 I hope you don't recommit found proxies to public repos.
